expyfun/_experiment_controller.py

```diff
@@ -933,6 +935,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             A time relative to which timestamping is done. Ignored if
             timestamp==False.  If ``None``, timestamps are relative to the time
             `listen_presses` was last called.
+        releases : False | bool
+            Whether key-release events should be returned as well.
```

Comment:
Also, I've been using [Google style docstrings](http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) as sphinx can autogenerate documentation from them. (But maybe that works with this format too?)